index.html

```diff
@@ -8,310 +8,339 @@
+      <div class="work-on">
+        <div class="all-about">
+          <h3 class="best">
+            <img src="icons/Rectangle55.png" alt="rectangl" /> Language
+          </h3>
+          <ul class="list">
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>Css</li>
+          </ul>
+        </div>
+        <div class="all-about">
+          <h3 class="best">
+            <img src="icons/Rectangle56.png" alt="rectangle" /> Frameworks
+          </h3>
+          <ul class="list">
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>Rspec</li>
+            <li>CapyBara</li>
+            <li>Selenium</li>
+          </ul>
+        </div>
+        <div class="all-about">
+          <h3 class="best">
+            <img src="icons/Rectangle57.png" alt="circle" /> Skills
+          </h3>
+          <ul class="list">
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+          </ul>
+        </div>
+      </div>
+    </section>
+    <section class="contact-form">
+      <div class="the-contact">
+        <h2 class="contact-p">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+
+        <form action="https://formspree.io/f/xnqwjqwy" method="post">
+          <div class="input__sec">
+            <div class="name">
+              <label>
+                <input
+                  type="text"
+                  id="full-name"
+                  name="name"
+                  placeholder="Full Name"
+                  required
+                  minlength="2"
+                  maxlength="30"
+                />
+              </label>
+            </div>
+            <p>
+              <label>
+                <input
+                  type="email"
+                  name="email"
+                  placeholder="Email address"
+                  required
+                />
+              </label>
+            </p>
+            <label>
+              <textarea
+                name="textarea"
                 required
-                maxlength="30"
-                placeholder="Your Email"
-            /></label>
-            <textarea
-              id="message"
-              cols="30"
-              rows="10"
-              maxlength="500"
-              placeholder="Write your message here"
-              required
-            ></textarea>
-            <div class="submit-button">
-              <button type="submit">Get in touch</button>
+                minlength="2"
+                maxlength="500"
+                placeholder="Hello Vistor! Glad to see you!"
+              ></textarea>
+            </label>
+            <div class="btn__container">
+              <button type="submit" name="submit" class="btn__submit">
+                Get in touch
+              </button>
```

Comment:
- [ ] As requested by the previous reviewer, please make sure that there is some margin between the `Get In Touch` button and the social icons.